tutorials/plantUml.md

```diff
@@ -278,3 +278,51 @@ package "Rule Of Thumb";{
 
 Explicitly define all symbols to avoid any potential layout mishaps.
 </box>
+
+### Using reference frames
+
```

Comment:
Perhaps we could add a short intro/context for this ref frame section to standardise with the others too!